notebooks/model_monitoring/solutions/model_monitoring_vertex.ipynb

```diff
@@ -0,0 +1,1294 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        input_uri="bq://mco-mm.bqmlga4.train"
</pre><p>Before we merge, let's try to replace this with our own BQ dataset or use CSV files. This will make us completely independent from someone else's BQ dataset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Done.</p>